app/js/helpers/jsonHelper.js

```diff
@@ -105,61 +104,4 @@ export class JSONHelper {
     });
     return colValues;
   }
-  
-  composeDescription(searchParameters) {
-    const newParameters = Object.assign({}, searchParameters);
-    let definitionKeys = Object.keys(newParameters);
-    if(definitionKeys.length === 0) {
-      return 'All Patients';
-    } else {
-      let label = '';
-      if (definitionKeys.indexOf('gender') >= 0) {
-        label = `${this.getGenderName(newParameters['gender'])} patients`;
-        delete newParameters.gender;
-        definitionKeys = Object.keys(newParameters);
```

Comment:
Great Observation @dkayiwa.
The JsonHelper is meant to help any of the search component generate the necessary parameters for a request/query. This deleted section generates the label for the query when a search is made using the patient attribute tab. What I have done is localize the generation of the label to the Patient attribute tab component, thereby leaving the JsonHelper to focus on only generating the necessary JsonParameter.